app/src/androidTest/java/com/duckduckgo/app/fire/WebViewCookieManagerTest.kt

```diff
@@ -17,66 +17,104 @@
 package com.duckduckgo.app.fire
 
 import android.webkit.CookieManager
+import android.webkit.ValueCallback
+import com.duckduckgo.app.CoroutineTestRule
+import com.duckduckgo.app.runBlocking
+import com.nhaarman.mockitokotlin2.*
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.withContext
-import org.junit.Assert.assertEquals
-import org.junit.Assert.assertTrue
 import org.junit.Before
+import org.junit.Rule
 import org.junit.Test
-import kotlin.coroutines.resume
-import kotlin.coroutines.suspendCoroutine
 
-@Suppress("RemoveExplicitTypeArguments")
-class WebViewCookieManagerTest {
+private data class Cookie(val url: String, val value: String)
 
-    private lateinit var testee: WebViewCookieManager
+class WebViewCookieManagerTest {
+    @get:Rule
+    @Suppress("unused")
+    val coroutineRule = CoroutineTestRule()
```

Comment:
Can you accept the lint suggestion to add the experimentation annotation? otherwise, there's a lot of yellow warnings that make it hard to see real lint issues